README.md

```diff
@@ -1,3 +1,33 @@
 # fasjson-client
 
 A python client library for the FASJSON API
+
+This client uses the bravado library to build dynamic api methods based on open-api specs (version 2.0): https://github.com/Yelp/bravado
+
+## Usage
+
+```
+c = Client.from_url('http://myspecurl.com', base_url='http://fasjson.example.test/fasjson' principal='admin@EXAMPLE.TEST')
```

Comment:
I think we should have the API server serve the spec at some known endpoint, and have the client automatically pull it from there. End users shouldn't have to know where we publish the spec file.

Reply:
right, this is is just a dummy readme file for now, we could host the spec definition in the root endpoint (/v1)

Reply:
I think we could have a default value that could be changed if needed but the value itself needs to be changed once we add this feature in the api